pom.xml

```diff
@@ -626,6 +632,19 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <version>0.7.5</version>
+        <configuration>
+          <excludes>
+            <!-- Simple autovalue classes -->
+            <exclude>org/ambraproject/wombat/model/Amendment.class</exclude>
```

Comment:
Now I know where to stash future untested code.  :)

Reply:
Unfortunately this doesn't actually work in the jacoco run on teamcity :(